docs/root/start/docker.rst

```diff
@@ -50,16 +55,30 @@ Build the Docker image using:
    $ docker build -t envoy:v1 .
 
 Assuming Envoy is configured to listen on ports ``9901`` and ``10000``, you can now start it
-with:
+in Docker with:
 
 .. code-block:: console
 
    $ docker run -d --name envoy -p 9901:9901 -p 10000:10000 envoy:v1
 
-Permissions for running Docker Envoy containers as a non-root user
+or in Podman (unsupported) with:
+
+.. code-block:: console
+
+   $ podman run -d --name envoy -p 9901:9901 -p 10000:10000 envoy:v1
+
+Root filesystem permissions for running Envoy in containers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Envoy container image can be run with the container's root filesystem mounted read-only.
+With Docker and Podman, this means using the ``--read-only`` option of the ``run`` command.
```

Comment:
```suggestion
For example, using Docker and Podman, you can use the ``--read-only`` option of the ``run`` command.
```

it doesnt necessarily mean using `run` - this should work equally with `up` or in a situation where you dont call the container directly (ie k8s/compose/etc)